async-contracts/ICofhe.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.25 <0.9.0;
+
+struct CiphertextKey {
+    bool     isTriviallyEncrypted;
+    uint8   uintType;
+    int32    securityZone;
+    uint256  hash;
+}
```

Comment:
I know it doesn't related to ur pr but please remove this from here and move this to TaskManager.sol

Reply:
https://github.com/FhenixProtocol/copro-task-manager-hardhat/pull/14